assets/js/main.js

```diff
@@ -0,0 +1,5 @@
+document.addEventListener("click", function (e) {
```

Comment:
Would it be better to add this event to each element that contains `dataset.dismissable`? Would these events be registered if you replaced content on the page with HTMX, how could we get around that?